manuscript/02-bestandteile-eines-modellgetriebenen-designs.md

```diff
@@ -1,177 +1,180 @@
-Geschäftslogik ist.  Unterteile ein komplexes Programm in Schichten.
-Entwickel innerhalb jeder Schicht ein Design, das kohärent ist und nur
-von den darunter liegenden Schichten abhängt.  Befolge die üblichen
-Architekturmuster, um eine lose Kopplung zu den darüber liegenden
+Benutzeroberfläche und sogar zur Anwendungslogik, die keine
+Geschäftslogik ist. Unterteile ein komplexes Programm in Schichten.
+Entwickle innerhalb jeder Schicht ein Design, das kohärent ist und nur
+von den darunterliegenden Schichten abhängt. Verwende gängige
+Architekturmuster, um eine lose Kopplung zu den darüberliegenden
 Schichten zu erreichen. Konzentriere den gesamten Code, der sich auf
-das Domänenmodell bezieht, in einer Schicht und isoliere es von der
-Benutzeroberfläche, der Anwendungscode und dem Infrastrukturcode.  Die
+das Domänenmodell bezieht, in einer Schicht und isoliere ihn von der
+Benutzeroberfläche, der Anwendungslogik und dem Infrastrukturcode. Die
 Domänenobjekte können sich auf die Umsetzung des Domänenmodells
 konzentrieren und sind frei von der Verantwortlichkeit, sich
 anzuzeigen, sich zu speichern, Aufgaben in der Anwendung zu verwalten
-usw. .  Dies ermöglicht es dem Modell, sich so zu entwickeln, dass es
-reichhaltig genug und klar genug sein kann, um grundlegendes Wissen
-über das Geschäft zu erfassen und umzusetzen.**
+usw. Dies ermöglicht es dem Modell, sich so zu entwickeln, dass es
+reichhaltig genug und klar genug sein kann, um grundlegendes 
+Geschäftswissen zu erfassen und umzusetzen.**
 
-Das Hauptziel dabei ist die Isolation.  Verwandte Muster, wie
-z.B. "Hexagonal Architecture", können ebenso gut oder besser dazu
+Das Hauptziel dabei ist die Isolation. Verwandte Muster, wie
+z.B. die "Hexagonal Architecture", können ebenso gut oder besser dazu
 beitragen, dass unsere Umsetzungen des Domänenmodells Abhängigkeiten
-von und Referenzen zu anderen Systembelangen vermeiden.
+auf und Referenzen zu anderen Systembelangen vermeiden.
 
 ## Entities {#entity}
 
-*Dt.: Entität, auch bekannt als Reference Objects*
+*dt.: Entitäten*
 
 Viele Objekte stellen eine Kontinuität und Identität dar und
 durchlaufen einen Lebenszyklus, obwohl sich ihre Attribute ändern
 können.
 
-Einige Objekte sind nicht in erster Linie durch ihre Attribute
-definiert.  Sie stellen einen Identität dar, die durch die Zeit und
-oft durch verschiedene Darstellungen durchläuft.  Manchmal muss ein
-solches Objekt mit einem anderen Objekt verglichen werden, obwohl die
-Attribute unterschiedlich sind.  Ein Objekt muss von anderen Objekten
-unterschieden werden, auch wenn sie die gleichen Eigenschaften haben
-können.  Eine falsche Identität kann zu Datenkorruption führen.
+Einige Objekte sind nicht in erster Linie über ihre Attribute
+definiert. Sie stellen eine Identität dar, welche die Zeit und oft 
+verschiedene Darstellungen durchläuft. Manchmal muss ein
+solches Objekt mit einem anderen Objekt übereinstimmen, obwohl die
```

Comment:
oder "stimmt überein" ohne "muss"? oder "zur Übereinstimmung gebracht werden"?

Reply:
"stimmt überein" passt für mich auch - ich wollte vor allem das "vergleichen" loswerden, weil es ja darum geht, dass die Objekte nicht nur verglichen werden müssen, sondern als "gleich" erkannt werden sollen.

Fixed in c9a050a03408b7f47c79f33f55d6884df7b17d8a